### PR TITLE
Blog Post: Long-Term Support For Tobin Harding

### PR DESCRIPTION
**Pull request by dergigi**
Approved by Tobin, let's see if we can publish this one today (Dec 26th)

**Build preview:**

- [/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin](https://os-website-git-tobin-lts-open-sats.vercel.app/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin)

**Social preview:**

Screenshot 2023-12-26 at 11 30 36

### CODE REVIEW

File: data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+of reviewing every public type and function, a meticulous process that often
+involves extensive discussions. This effort promises a stable API with minimal
+changes expected for at least a year.
+
+On top of that, he’ll persist in iterating on the API to maintain its alignment
+with [Rust API design principles](https://github.com/rust-lang/api-guidelines),
+ensuring user-friendliness for both experienced Bitcoin developers and
+newcomers. Complementing this, Tobin is dedicated to delivering clear and
+accessible documentation. He’s also simultaneously working on a v2 of his
+[rust-PSBT](https://github.com/tcharding/rust-psbt) implementation.
```
Comment (tcharding) [inline data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx:40]:
```suggestion
newcomers. Complementing this, Tobin is aiming to delivering clear and
accessible documentation. He’s also simultaneously working on a new
[rust-PSBT](https://github.com/tcharding/rust-psbt) implementation
(PSBT v0, v2, and an API overhaul).
```

File: data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+[electrs](https://github.com/romanz/electrs).
+
+Tobin's overarching goal is to release version 1.0.0 for
+[rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1),
+[rust-bech32](https://github.com/rust-bitcoin/rust-bech32),
+[bitcoin_hashes](https://github.com/rust-bitcoin/bitcoin_hashes) and
+[rust-bitcoin](https://github.com/rust-bitcoin). He is also in the final stages
+of reviewing every public type and function, a meticulous process that often
+involves extensive discussions. This effort promises a stable API with minimal
+changes expected for at least a year.
```
Comment (tcharding) [inline data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx:33]:
```suggestion
Tobin's overarching goal is to get `rust-bitcoin` to a version 1.0.0 release.
On the way, version 1.0.0 releases of 
[rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1),
[rust-bech32](https://github.com/rust-bitcoin/rust-bech32),
[bitcoin_hashes](https://github.com/rust-bitcoin/bitcoin_hashes) and others.
He is also focused on the effort of reviewing every public type and function, a
meticulous process that often involves extensive discussions. This effort promises
a stable API with minimal changes that you guys can rely on.
```

Comment (tcharding) [reply data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx:33]: My whitespace may not be correct, may need changing to suit the rest of the document.

**Review by tcharding — APPROVED** [review]
Look great, thanks. Its hard reading about oneself, please feel free to ignore my suggestions if the original or some other form is deemed better. Thanks.

**Comment by tcharding** [comment]
I checked all the links in the original, the `rust-bitcoin` one is changed in my review suggestions to include the repo name.

File: data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx (hunk 1 above)
Comment (dergigi) [reply data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx:41]: I think that's too much detail for a general audience, I'll leave the original as is.

**Review by Arvin21M — APPROVED** [review]
All good. No noticeable grammatical and punctuational issues. Links load and point to the right places. Happy to see Rust making its way through Bitcoin.

**Pull request merged**